sdk/python/tests/test_cli_redis.py

```diff
@@ -29,11 +29,11 @@ def test_basic() -> None:
                 f"""
         project: {project_id}
         registry: {data_path / "registry.db"}
-        provider: redis
+        provider: local
         offline_store:
             type: bigquery
         online_store:
-            redis_type: redis
+            type: redis
```

Comment:
wait, is this a bug? I believe there is a `type` and a `redis_type`, and they are different.

Reply:
This is needed cause redis provider assumed the type of the online store to be of redis type - but now there's no redis provider so we need to mention this type explicitly, and use the default `redis_type`, which is `redis`.